src/components/BigButton/Home.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import BigButton from './BigButton';
+
+test('renders learn react link', () => {
```

Comment:
It's ok to use jest but I would prefer test react components with react testing library

Reply:
Jest is used as testrunner, but the testing process is done with react testing library.